src/js/builtins.d.ts

```diff
@@ -99,6 +99,14 @@ declare function $getMapIteratorInternalField(): TODO;
 declare function $getSetIteratorInternalField(): TODO;
 declare function $getProxyInternalField(): TODO;
 declare function $idWithProfile(): TODO;
+/**
+ * True for `JSCell`s. That is, this is roughly equivalent to this JS code:
```

Comment:
```suggestion
 * True for `JSCell` that are are object-like. That is, this is roughly equivalent to this JS code:
```

Reply:
note: I found that this was returning `true` for functions as well